commerce/bookstore/python/loaders.py

```diff
@@ -0,0 +1,151 @@
+#
+# Copyright (C) 2022 Vaticle
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+import random, uuid
+from typedb.client import TypeDB, SessionType, TransactionType
+import config
+
+debug = False
```

Comment:
We introduced a command-line argument `verbose` to replace the `debug` field: let's now get it into this file.

It's currently used in two places: `OrderInput.load` and the function `random_books`. This presents a slight awkwardness: how do we avoid just throwing `verbose` around the whole application?

For `Loader`s we have a couple of choices to make, feel free to make a decision on this:

1) Do we make `verbose` a property of `Loader`, or a parameter in `load`?
2) Do we add `verbose` to the base class `Loader`, or specifically to `OrderInput`?

For `random_books`, given that it's just a single method, it's probably OK to just add a `verbose` parameter. If we had 3 similar methods, I'd say that would be the threshold to trigger a refactor (eg introducing a `RandomDataGenerator` class.)

Reply:
Implemented this in https://github.com/vaticle/typedb-examples/pull/143/commits/1964bc9fef1a1f6eafc65f859fbebef16a878199